packages/picasso-lab/src/TextEditor/TextEditor.tsx

```diff
@@ -72,25 +98,92 @@ export const TextEditor = forwardRef<HTMLDivElement, Props>(function TextEditor(
 ) {
   const editorRef = useRef<Quill>()
   const classes = useStyles()
+  const [selected, setSelected] = useState<string>(OPTIONS[0].value)
+  const [isActive, setIsActive] = useState<ToolbarActions>({
+    bold: false,
+    italic: false,
+    unorderedList: false,
+    orderedList: false
+  })
 
   useInitEditor(editorRef, { id, placeholder })
   useHandleChangeEvent(editorRef, { onChange })
   useHandleChangeFromController(editorRef, { value })
   useDisableEditor(editorRef, { disabled })
   useHandleAutofocus(editorRef, { autofocus })
 
+  const handleChange = (event: React.ChangeEvent<{ value: string }>) => {
+    setSelected(event.target.value)
+  }
+
+  const toggleActive = (key: ToolbarKey) => () =>
+    setIsActive((prev: ToolbarActions) => ({ ...prev, [key]: !prev[key] }))
+
   return (
-    <Typography
-      as='div'
-      variant='body'
-      color='dark-grey'
-      size='medium'
-      className={cx(classes.root, className)}
-      data-testid={dataTestId}
-      id={id}
-      ref={ref}
-      style={style}
-    />
+    <Container className={classes.editorWrapper}>
+      <Container id={`${id}toolbar`} className={classes.qlToolbar}>
+        <Container className={classes.qlFormats}>
+          {
+            // TODO: add styling to options
+          }
+          <Select
+            onChange={handleChange}
```

Comment:
To me it would be helpful to have more domain specific names:
```suggestion
            onChange={handleFormattingChange}
```